README.md

```diff
@@ -120,7 +120,7 @@ A few sources are searched for guessing `python_requires`:
 - the existing `python_requires` setting itself
 - `envlist` in `tox.ini` if present
 - python version `classifiers` that are already set
-- the `--min-py-version` argument (currently defaulting to `3.7`)
+- the `--min-py-version` argument (currently defaulting to `3.8`)
```

Comment:
```suggestion
- the `--min-py-version argument
```

Like this?

(actually, isn’t this `—-min-py3-version` still?)

Reply:
`--min-py3-version` is the deprecated alias to `--min-py-version`

and yeah this looks fine -- can you amend it into your current commit?

Reply:
Done! I thought from seeing #179 that `--min-py-version` hadn't been added yet, but I see it is there.